lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -0,0 +1,268 @@
+defmodule PhxToolsWeb.PhxToolsComponents do
+  @moduledoc false
+
+  use PhxToolsWeb, :html
+
+  alias Phoenix.LiveView.Utils
+  alias PhxToolsWeb.PhxToolsLive.Icons
+
+  @type assigns :: map()
+  @type rendered :: Phoenix.LiveView.Rendered.t()
+
+  @spec command_select_button(assigns()) :: rendered()
+  def command_select_button(assigns) do
+    ~H"""
+    <.os_link_button
+      id={if @live_action == :macOS, do: "Linux", else: "macOS"}
+      href={if @live_action == :macOS, do: ~p"/Linux", else: ~p"/macOS"}
+      os_name={if @live_action == :Linux, do: "macOS", else: "Linux"}
+      class="px-3 flex justify-center"
+    />
```

Comment:
Seems like this could be done inside the component with `Atom.to_string(@live_action)` or "#{@live_action}". Maybe pass the os attribute instead:
```suggestion
    <.os_link_button
      class="px-3 flex justify-center"
      os={Atom.to_string(@live_action)}
    />
```
and then use `@os` inside the component.

Reply:
I just realised the function calling this may actually not be necessary anymore since I can use this directly.